storage/buckets/buckets_test.go

```diff
@@ -153,7 +194,10 @@ func TestKMS(t *testing.T) {
 
 	kmsKeyName := fmt.Sprintf("projects/%s/locations/%s/keyRings/%s/cryptoKeys/%s", tc.ProjectID, "global", keyRingID, cryptoKeyID)
 	if err := setBucketDefaultKMSKey(ioutil.Discard, bucketName, kmsKeyName); err != nil {
-		t.Fatalf("setBucketDefaultKmsKey: failed to enable default kms key (%q): %v", kmsKeyName, err)
+		t.Fatalf("setBucketDefaultKmsKey: failed to enable default KMS key (%q): %v", kmsKeyName, err)
+	}
```

Comment:
Let's check that the key was actually set (and then that it was actually removed after the remove sample).

Reply:
Done